pkgs/servers/http/apache-modules/mod_dnssd/default.nix

```
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
   buildInputs = [ apacheHttpd avahi apr ];
 
   patches = [ (fetchpatch {
-    url = "http://bazaar.launchpad.net/~ubuntu-branches/ubuntu/vivid/mod-dnssd/vivid/download/package-import%40ubuntu.com-20130530193334-kqebiy78q534or5k/portforapache2.4.pat-20130530222510-7tlw5btqchd04edb-3/port-for-apache2.4.patch";
+    url = "https://bazaar.launchpad.net/~ubuntu-branches/ubuntu/vivid/mod-dnssd/vivid/download/head:/debian/patches/port-for-apache2.4.patch";
```

```suggestion
    url = "https://bazaar.launchpad.net/~ubuntu-branches/ubuntu/vivid/mod-dnssd/vivid/download/10/debian/patches/port-for-apache2.4.patch";
```
To ensure it doesn't break in the future specify the revision explicitly.